docs/content/developers/design-keuzes.md

```diff
@@ -172,3 +172,21 @@ de fout-responses (zie ook #130) correct teruggegeven worden.
 
 Noot 2: authenticatie en authorisatie worden in een later stadium uitgewerkt.
 Het is bekend dat hier goed over nagedacht moet worden.
+
+
+## Besluitenregistratie (BRC)
+
+Er komt een aparte besluitenregistratie (naast ZRC, ZTC en DRC) om besluiten
+in vast te leggen. De motivatie hiervoor is dat besluiten een bestaansrecht
+hebben onafhankelijk van zaken: niet elk besluit ontstaat gedurende de
+uitvoering van een zaak (denk aan raadsbesluiten).
+
+Hierbij lopen we wat vooruit, maar volgen we wel data-bij-de-bron principes uit
+Common Ground.
+BESLUITTYPE laten we vooralsnog in de ZTC. Nader bepaald moet gaan worden hoe we hiermee omgaan, zoals onderbrengen in de BRC, aparte type-componenten cq. api's e.d.
+
+Dit betekent dat er voor BESLUIT een informatiemodel moet komen, ontrokken aan
+RGBZ resp. ImZTC. BESLUIT krijgt een optionele relatie naar ZAAK.
+
+De verwachting is dat er later vergelijkbare designkeuzes gemaakt worden voor
+andere resources, bijvoorbeeld klantcontacten, besluittype, etc.
```

Comment:
De paragraaf hierboven spreekt dit direct tegen, toch?

Reply:
Ik neem aan dat je op de paragraaf 'Inputvalidatie' doelt. Ik zie het verband niet.

Reply:
Ik doel op 

> BESLUITTYPE laten we vooralsnog in de ZTC. Nader bepaald moet gaan worden hoe we hiermee omgaan, zoals onderbrengen in de BRC, aparte type-componenten cq. api's e.d.

Reply:
Het is wel een aparte keuze maar had daar wellicht weggelaten mogen worden.

Ik ben tevreden met dat voor nu het BR alleen BESLUIT bevat.

Reply:
> De verwachting is dat er later vergelijkbare designkeuzes gemaakt worden voor andere resources, bijvoorbeeld klantcontacten, besluittype, etc. 


Aha, over die zin gaat het. Daar zou ik inderdaad 'besluittype' niet noemen. 
En dan ook even in de zin daarboven ' resp. ImZTC' er uit halen. Dat stond er vanwege het besluittype.

Reply:
Ah, ik volgde em ook niet maar aangepast.

Reply:
@ArjanKloosterboer als je via het 'Files changed' tabje kijkt naar de commentaar, dan zie je bij welke lijn de commentaar hoort. Ik houd de commentaar altijd zo dicht mogelijk bij de betreffende lijn. Waarschijnlijk is dat nog even wennen.